engine/audio/src/audiocapture.cpp

```diff
@@ -151,10 +153,10 @@ double AudioCapture::fillBandsData(int number)
     // representing all the frequencies from 0 to m_sampleRate Hz.
     // I will just consider 0 to 5000Hz and will calculate average magnitude
     // for the number of desired bands.
-    double maxMagnitude = 0;
+    double bandMagnitude = 0;
 #ifdef HAS_FFTW3
-    unsigned int i = 0;
-    int subBandWidth = ((m_captureSize * SPECTRUM_MAX_FREQUENCY) / m_sampleRate) / number;
+    unsigned int i = 1; // skip DC bin
```

Comment:
This should definitely be 1 because the zero bin contains the DC offset and there is no point finding its power.